packages/calcite-components/src/components/autocomplete-item-group/autocomplete-item-group.scss

```diff
@@ -0,0 +1,61 @@
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden using the component's tag as selector.
+ *
+ * @prop --calcite-autocomplete-background-color: Specifies the background color of the component.
+ * @prop --calcite-autocomplete-border-color: Specifies the border color of the component.
+ * @prop --calcite-autocomplete-text-color: Specifies the text color of the component.
+ */
+
+.scale--s {
+  @apply text-n2h;
+  --calcite-internal-autocomplete-item-group-spacing-unit-l: theme("spacing.2");
+  --calcite-internal-autocomplete-item-group-spacing-unit-s: theme("spacing.2");
```

Comment:
Nitpick : can we use one variable here `--calcite-internal-autocomplete-item-group-spacing-unit`. Applies to all scales.